feature-libs/checkout/b2b/components/checkout-review-submit/checkout-review-submit.component.html

```diff
@@ -131,10 +131,33 @@ <h2 class="cx-review-title d-none d-lg-block d-xl-block">
   <!-- DELIVERY MODE SECTION -->
   <ng-template #deliveryMode>
     <div class="cx-review-summary-card cx-review-card-shipping">
-      <cx-card
-        *ngIf="deliveryMode$ | async as deliveryMode"
-        [content]="getDeliveryModeCard(deliveryMode) | async"
-      ></cx-card>
+      <!-- TODO:(CXINT-1850) for next major release remove feature level -->
```

Comment:
Please create and link a new JIRA ticket number.
As a description please add paths to the files which are affected by your feature level directive.
Also mark delivery version of the ticket as 7.0.

Here you can find an example ticket: https://jira.tools.sap/browse/CXSPA-3090

Reply:
sure,

A new ticket https://jira.tools.sap/browse/CXINT-2309 has been created and the comments have been updated.